backend/src/routes/facilities.js

```diff
@@ -31,3 +31,11 @@ router.get('/:facilityId/licences', passport.authenticate('jwt', { session: fals
   validationResult(req).throw()
   return getFacilityLicences(req, res)
 })
+
+/**
+ * Update an Facilities Primary Contact
+ */
+router.put('/:facilityId', passport.authenticate('jwt', { session: false }), isValidBackendToken, [param('facilityId', 'URL param: [facilityId] is required').not().isEmpty()], (req, res) => {
+  validationResult(req).throw()
+  return updateFacilityPrimaryContact(req, res)
```

Comment:
should we change updateFacilityPrimaryContact() to be more generic (i.e.: updateFacility --> pass an object with key/value pair of all changes)? so we can reuse this function for update Facility Information. I think the implementation can be similar to updateApplication